docs/BUILD

```diff
@@ -1 +1,8 @@
-exports_files(["rule_template.vm"])
+load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+
+stardoc(
+    name = "docs",
+    out = "rule.md",
+    input = "//detekt:defs.bzl",
+    rule_template = "//docs:rule_template.vm",
+)
```

Comment:
Had to move the Stardoc configuration since the `rules_detekt_toolchains` macro falls back to the default toolchain in the `detekt/BUILD` file. Which means that a different workspace tries to `load` Stardoc which does not exist (and we don‘t want users to declare it).